cli/src/config.rs

```diff
@@ -0,0 +1,143 @@
+use color_eyre::Result;
+use hyper::{http::HeaderName, Method};
+use serde::Deserialize;
+use serde_with::{serde_as, DisplayFromStr};
+use std::{fs::File, io::BufReader, net::SocketAddr, path::PathBuf, time::Duration};
+use tower_http::cors::CorsLayer;
+
+pub type Timeout = u64;
+
+/// Server Configuration passed on initialization
+#[serde_as]
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    /// Server address <host:port>
+    pub address: SocketAddr,
+
+    /// Cors configuration
+    pub cors: Cors,
+
+    /// Max Time to Responce, in milliseconds
+    pub request_timeout: Timeout,
+
+    /// [Stub] File to save logs
+    pub log_file: Option<PathBuf>,
```

Comment:
In production, it's considered to be a good practice to use logs rotation. Please, add the appropriate parameters for that: number of log files and max size of a single log file